cobc/tree.c

```diff
@@ -2883,43 +2888,44 @@ emit_precedence_error (const int preceding_idx, const int following_idx)
 static int
 valid_char_order (const cob_pic_symbol *str, const int s_char_seen)
 {
-	const int	precedence_table[24][24] = {
+	const int	precedence_table[25][25] = {
 		/*
 		  Refer to the standard's PICTURE clause precedence rules for
 		  complete explanation.
 		*/
 		/*
-		  B  ,  .  +  +  + CR cs cs  Z  Z  +  + cs cs  9  A  S  V  P  P  1  N  E
+		  B  ,  .  +  +  + CR cs cs  Z  Z  +  + cs cs  9  A  L  S  V  P  P  1  N  E
 		  0           -  - DB        *  *  -  -           X
 		  /
 		*/
-		{ 1, 1, 1, 0, 1, 0, 0, 1, 0, 1, 1, 1, 1, 1, 1, 1, 1, 0, 1, 0, 1, 0, 1, 0 },
```

Comment:
Which part of the GCOS documentation did you found the precedence for `L` in?

Reply:
The precedence rules for character L that can be found on page 287 at https://support.bull.com/ols/product/system/gcos7/gcos7-com/g7-dps7000/doc-com/docf/g/47X257TN23-mar2005/doc/05ula04.pdf
The rules for the other character types are as before the change.